x-pack/metricbeat/module/oracle/connection.go

```diff
@@ -6,62 +6,54 @@ package oracle
 
 import (
 	"database/sql"
+	"fmt"
 
 	"github.com/godror/godror"
 
 	"github.com/elastic/beats/v7/metricbeat/mb"
-	"github.com/elastic/beats/v7/metricbeat/mb/parse"
-
-	"github.com/pkg/errors"
 )
 
 // ConnectionDetails contains all possible data that can be used to create a connection with
 // an Oracle db
 type ConnectionDetails struct {
-	Username string   `config:"username"`
-	Password string   `config:"password"`
-	Hosts    []string `config:"hosts"    validate:"required"`
+	Username string `config:"username"`
+	Password string `config:"password"`
 }
 
-// HostParser parsers the host value as a URL
-var HostParser = parse.URLHostParserBuilder{
-	DefaultScheme: "oracle",
-}.Build()
-
-func init() {
-	// Register the ModuleFactory function for the "oracle" module.
-	if err := mb.Registry.AddModule("oracle", newModule); err != nil {
-		panic(err)
+// HostParser parses host value
```

Comment:
Could you please add a more detailed comment here?
like `HostParser` parses host and other information from `connectionString` and giving example of `connectionString` and exact output.